src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Ball[] ball = new Ball[3];
+
+        for (int i = 1; i <= 3; i++) {
+
+            ball[i - 1] = new Lottery().getRandomBall();
+            System.out.println("Ball #" + i + " was created in " + ball[i - 1].getColor()
+                    + " color with number of " + ball[i - 1].getNumber());
+        }
+
     }
```

Comment:
```suggestion
        Ball[] balls = new Ball[3];
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            balls[i] = lottery.getRandomBall();
            System.out.println("Ball #" + i + " was created : " + balls[i]);
        }
    }
```

Reply:
Please explain why this next was changed. How this part " was created : " + balls[i]); can print anything without calling to some method?

for (int i = 0; i < 3; i++) and
System.out.println("Ball #" + i + " was created : " + balls[i]);